cmds/boot/boot.go

```diff
@@ -0,0 +1,451 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// boot allows to handover a system running linuxboot/u-root
+// to a legacy preinstalled operating system by replacing the traditional
+// bootloader path
+
+//
+// Synopsis:
+//	boot
+//
+// Description:
+//	If returns to u-root shell, the code didn't found a local bootable option
+//
+// Notes:
+//	The code is looking for boot/grub/grub.cfg file as to identify the
+//	boot option.
+//	The first bootable device found in the block device tree is the one used
+//	Windows is not supported (that is a work in progress)
+//
+// Example:
+//	boot -v 	- Start the script in verbose mode for debugging purpose
+
+package main
+
+import (
+	"bufio"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"github.com/u-root/u-root/pkg/kexec"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"flag"
+	"syscall"
+)
+
+const (
+	bootableMBR = 0xaa55
+	signatureOffset = 510
+)
+
+var verbose bool
+type options struct {
+        verbose bool
+}
```

Comment:
couple options here. 
var options struct {
verbose bool
}
get rid of var verbose bool and see below for possible change to main.

Reply:
OR just have
var verbose bool

since maybe it's easier to say 
verbose
not
options.verbose